Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,25 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Enter first name: ");
+
+            var firstName = Console.ReadLine();
+
+            Console.WriteLine("Enter last name: ");
+            var lastName = Console.ReadLine();
+
+            Console.WriteLine("Enter age: ");
+            var age = Int32.Parse(Console.ReadLine());
+
+            Console.WriteLine("Enter weight (in kg): ");
+            var weight = Double.Parse(Console.ReadLine());
+            Console.WriteLine("Enter weight (in cm): ");
```

Comment:
Here a blank line is needed, so separate weightr and height. Also copy pasted too much, Enter weight is twice.